misc/userscripts/cast

```diff
@@ -133,17 +133,23 @@ echo "jseval -q $(printjs)" >> "$QUTE_FIFO"
 
 tmpdir=$(mktemp -d)
 file_to_cast=${tmpdir}/qutecast
+program_=$(which castnow)
+
+if [[ "${program_}" == "" ]]; then
```

Comment:
I think you want `[ -z "${program_}" ]`

Reply:
I think that `[` is `sh` only and that the proper way of writing that in `bash` is using the double brackets `[[`. According to [bashFAQ][faq], I could use `[[ -z`, however, I think that it's equivalent of writing what I did. Isn't it?

[faq]: http://mywiki.wooledge.org/BashFAQ/031

Reply:
I personally write everything with as low of a compatibility level as possible (so I prefer `[]` over `[[]]`). In addition, `==` isn't posix inside `[]`, so I always prefer `-z` over `= ""`, to try to avoid using `==` by mistake. (it's very common to see `[ -z "$VAR" ]` to check to see if a variable is set portably.

However, since I noticed we are requesting bash specifically (and we can probably assume a recently installed version), so I guess this is fine.

Reply:
Yes. It was in fact working before I switched the installation location of `castnow` program. That's how I realized there was an error in my code.